ipfs-cluster-service/main.go

```diff
@@ -342,8 +344,12 @@ func setupLogging(lvl string) {
 
 func setupAllocation(strategy string) (ipfscluster.Informer, ipfscluster.PinAllocator) {
 	switch strategy {
-	case "disk", "reposize":
-		return disk.NewInformer(), ascendalloc.NewAllocator()
+	case "disk", "disk-freespace":
+		informer := disk.NewInformer()
+		return informer, descendalloc.NewAllocator()
```

Comment:
@hsanjuan It might be nice to have a way for the `Informer` to specify which `Allocator` it should use based on the metric (not sure whether there's a good reason to keep those decoupled like they are now).